benchmark/benchmarkTests.go

```diff
@@ -100,26 +101,36 @@ func SociFullRun(
 	defer sociProcess.StopProcess()
 	sociContainerdProc := SociContainerdProcess{containerdProcess}
 	b.ResetTimer()
+	log.G(ctx).WithField("test_name", testName).WithField("uuid", testUUID).WithField("benchmark", "Test").WithField("event", "Start").Infof("Start Test")
```

Comment:
You should make a logger with the common attributes once.

https://github.com/sirupsen/logrus#example

Reply:
Thanks updated in rev.  Using pattern
```
	ctx = log.WithLogger(ctx, log.G(ctx).WithField("test_name", testName))
	ctx = log.WithLogger(ctx, log.G(ctx).WithField("uuid", testUUID))
......
        log.G(ctx).WithField("benchmark", "Test").WithField("event", "Start").Infof("Start Test")
```